models/event.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const eventSchema = new Schema({
+  eventName: { 
+    type: String,
+    trim: true,
+    unique: true, // We would want a '_id' right? Is this what generates it? This would cause event names to be unique which we don't necessarily want...?
```

Comment:
Yeah the collection itself auto-generates an  _id for each document by default, and the unique property here would reject any events with the same name. I think we can remove that one, unless we want to make event names not repeatable.

Reply:
Nice, yes I think we can remove the unique key